### PR TITLE
Add docker command wrapper.

### PR DESCRIPTION
**Pull request by scholtalbers**
For example, to change the current group to docker, which is needed when the galaxy user belongs to (too) many groups.

### CODE REVIEW

File: lib/galaxy/web/base/interactive_environments.py [hunk 1]
```diff
@@ -189,7 +190,10 @@ def docker_cmd(self, env_override={}, volumes=[]):
              temp_dir,
              volume_str,
              self.attr.viz_config.get("docker", "image"))
-
+        cmd_wrapper = self.attr.viz_config.get("docker", "command_wrapper")
+        if cmd_wrapper != '' and '{cmd}' in cmd_wrapper:
+            cmd = cmd_wrapper.format( cmd=cmd )
+        return cmd
```
Comment (nsoranzo) [inline lib/galaxy/web/base/interactive_environments.py:196]: Please add a blank empty line here to fix the Travis flake8 error.

**Comment by hexylena** [comment]
@scholtalbers can you not use the `command` option in the IE config? That's what Bjoern uses to do `sudo docker`. E.g. https://gist.github.com/remimarenco/2605ab9fad6cc964983d#file-ipython-ini-L12

**Comment by bgruening** [comment]
Same question as @erasche  :)

**Comment by hexylena** [comment]
@bgruening we chatted about this in IRC, Jelle needs to use the `sg` command to switch primary group. http://linux.about.com/od/commands/a/Example-Uses-Of-The-Command-Sg.htm 

I'm going to +1 and merge his PR, and then I think we should roll `command`, `command_inject`, and `command_wrapper` into a single `command` option.

**Comment by hexylena** [comment]
Okay, LGTM. As soon as Travis passes I'll merge this and get started on a unified version.

**Pull request merged**